### PR TITLE
Develop misc fixes

### PR DESCRIPTION
**Pull request by madazaharia**

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 SPEC CHECKSUMS:
-  Alamofire: e266cf991b933498df791e645e0d8ac294492153
+  Alamofire: 16ce2c353fb72865124ddae8a57c5942388f4f11
   SwiftKeychainWrapper: 38952a3636320ae61bad3513cadd870929de7a4a
 
-PODFILE CHECKSUM: a2d502eb825b0bdbdca99e5ebfc27758148a0767
+PODFILE CHECKSUM: 4c2ab9b6f149352ba82bdd4b4050232e222a4a12
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.5.3
```
Comment (ccorneliu) [inline Podfile.lock]: A facut si @CristiHabliuc un Cocoapods update aici: #52 Posibil sa faca conflict la merge :D. Poate ar ajuta si ca toti sa folosim aceeasi versiune: the latest now.

**Comment by CristiHabliuc** [comment]
@ccorneliu Salutare! Long time no speak. Putem face un `.xcconfig` pentru setari din astea secrete/diferite per environment. Un articol f bun despre cum se poate face asta e [aici](https://www.matrixprojects.net/p/xcconfig-for-shared-projects/). Il pot implementa eu daca vreti. Cred ca mai bine fac si un issue

**Comment by CristiHabliuc** [comment]
Am facut issue: #55

**Comment by ccorneliu** [comment]
Thanks, @CristiHabliuc ! Good idea and thanks for the article!

**Comment by RaduCStefanescu** [comment]
Hi @madazaharia unfortunately this pull request has many outstanding issues that were not addressed for quite some time. I will close it now. Thank you for your work and effort.

**Pull request closed without merging**